package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "js-weather-app",
+  "version": "1.0.0",
+  "description": "js-the-weather-app is a simple javascript application that consults the Open Weather API and shows you the information about the weather of the location you query.",
+  "private": true,
+  "scripts": {
+    "build": "webpack",
+    "watch": "webpack --watch"
+  },
```

Comment:
- [ ] Nice job adding webpack to your scripts, but we also need to inform in which [mode](https://webpack.js.org/configuration/mode/) Webpack will run (there are differences, check the link provided). Your scripts should look something like this:
```javascript
"scripts": {
    "dev": "webpack --mode development --watch",
    "build": "webpack --mode production",
```

- [ ] You also should update your README file to inform about the scripts here at the package.json file so other people can test your app.

**OPTIONAL**
- [ ] Add `webpack-dev-server` dependency to your project and create an additional script to be used in development stages. 
```javascript
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open"
```
This will make it easier for other people to test your app since it only needs to run the script and there will be no need to open the index.html file in the browser.